fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java

```diff
@@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
     }
 
     @Test
+    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
+        final String prop = "premis:hasEventRelatedObject";
+        final String uri = "http://localhost:8080/rest/1";
```

Comment:
The relPath argument is the property name, not the value.  I copied the parameters javadoc from the getProperty() method for consistency, but I could update this to be clearer that it's the property name in abbreviated format "premis:hasEventRelatedObject", not the full URI.